### ISSUE TITLE
Voxel highlights in recipies too hard to see

### ISSUE THREAD

**Issue by Jack7D1** [anegostudios/VintageStory-Issues#2658]
### Game Version

v1.18.1

### Platform

Linux

### Modded

Vanilla

### SP/MP

Singleplayer

### Description

Highlighted voxels in recipe too hard to see. I remember the highlights being very clear to see.

No graphics settings appear to affect, however turning gamma all the way down, and during night time the highlights are more visible.
Mods have no effect. Possibly linux exclusive? Unable to tell.

Please let me know if there's a setting I missed that can change this.
Don't be afraid to ask for more info.

### How to reproduce

Start knapping/Clayforming/Anvil project

### Screenshots

![image](https://user-images.githubusercontent.com/59462654/234150004-c30929d9-d199-4533-9e8b-6649bb480862.png)
![image](https://user-images.githubusercontent.com/59462654/234150037-c89e1aae-edbc-4dae-8ad7-ed9f514d3e46.png)
![image](https://user-images.githubusercontent.com/59462654/234150468-f8e480e9-7a3f-4eb2-9b08-e8e1b6112c76.png)

### Logs

N/A

**Comment by daretmavi** [comment]
Maybe some configuration for voxel line width could be implemented.
It is really annoying, not to see what has to be done.

**Comment by Jack7D1** [comment]
I'm sorry I couldn't provide more info.

**Comment by Jack7D1** [comment]
After installing and testing 1.18.5 I notice this bug no longer occurs and the highlights are back to their usual visibility.
This wasn't listed in the changelog which is why I hadn't checked sooner, but perhaps its the mesa_gl install? Or a driver update on my end? I'm unsure what fixed it but it is now working again. If someone else is still experiencing it after updating to 1.18.5 and updating drivers please comment and will re open.

Otherwise, thanks for (maybe?) fixing this issue!

**Comment by daretmavi** [comment]
No change for me.
Can you write, which mesa version do you use?

**Comment by Jack7D1** [comment]
4.6 Mesa 23.1.1

**Comment by daretmavi** [comment]
I tried 4.6 Mesa 23.1.2 - kisak-mesa PPA (https://launchpad.net/~kisak/+archive/ubuntu/kisak-mesa?ref=itsfoss.com)

Looks like mesa was the reason. Lines are good visible with mesa 23.1.2.

One weird think is, game freezes if `mesa_glthread=true` is not set. This is exact opposite to Mesa 22.2.5.

**Comment by Jack7D1** [comment]
Well that settles it then! Thanks a ton!
